fem/ceed/interface/coefficient.hpp

```diff
@@ -33,6 +32,8 @@ class GridFunction;
 namespace ceed
 {
 
+#ifdef MFEM_USE_CEED
+
```

Comment:
Is there a reason this `#ifdef` was moved to an inner scope?

Reply:
I didn't realize I had done that, good catch, reverted in [460fa61](https://github.com/mfem/mfem/pull/2952/commits/460fa618fbe82edb38a52af159e2df2396a11f78).